sdk/synapse/azure-synapse-artifacts/CHANGELOG.md

```diff
@@ -1,11 +1,56 @@
 # Release History
 
-## 0.18.1 (Unreleased)
+## 0.19.0 (2023-05-10)
 
 ### Features Added
```

Comment:
please regenerate with new codegen version https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-with-tools#update so that setup.py could be updated.

Reply:
regenerated